_drafts/2017-05-31-product-goals.md

```diff
@@ -0,0 +1,53 @@
+If you look closely at our [repositories][ce0fc2d5], you may see that componentization has already started. What was one big chunk of code in the old sensenet ECM, now consists of 2 separate packages. There are at least 4 more to come. Once we're done, we'll aim to release hotfixes every other week and have feature releases at least once in a quarter (depending on the size of the feature).
+![Sensenet Components](https://github.com/SenseNet/sn-resources/raw/master/images/sn-components/sn-components_services.png)
+
+  [ce0fc2d5]: https://github.com/SenseNet/ "Sensenet"
+
+If you'd like to know more about sensenet components, check [here][36459043].
+
+  [36459043]: http://community.sensenet.com/docs/sensenet-components/ "sensenet ECM components"
+
+#### Product identity redesign
+The *sensenet* brand was long neglected, all of us needed something new. Something modern. Our site had obsolete information and our [documentation][8206ff40] was losing its edge too. As you can see, this too, is changing. New logo, new site design and features. New communication: sensenet ECM is a *development platform* that gives you the perfect tools for building [intranets][8f9b4f00] or [content repositories][bc64591a].
+
+  [8f9b4f00]: https://www.sensenet.com/for-customers/use-cases/intranet "Intranets"
+  [bc64591a]: https://www.sensenet.com/for-customers/use-cases/content-repository "Content Repository"
+  [8206ff40]: http://wiki.sensenet.com/Main_Page "Wiki.sensenet.com"
+
+We documented part of our transition to the new site in a previous blog post [here][dbea5825].
+
+  [dbea5825]: http://community.sensenet.com/blog/2016/10/27/redesigning-sn "Redesigning SN"
+
+#### Measure *all the things*
+The team wanted to know if we were developing the right features for the right set of people. We needed better channels for communication and lots of data. Right now we aggregate a fair amount of [GitHub][bebd2231] statistics with [NuGet][80c6156c], [npm][ffdb2e87], [Twitter][0c3b4e3a], [Facebook][b175e3d7] and Google Analytics data so we can have a clearer picture of our "community health". The next step is product-related usage statistics: it is crucial to know how people use our product and how can we create better user experiences.
+
+  [bebd2231]: https://github.com/SenseNet/sensenet "sensenet Github"
+  [80c6156c]: nuget.org/profiles/sensenet "sensenet Nuget"
+  [ffdb2e87]: https://www.npmjs.com/org/sensenet "sensenet on npm"
+  [0c3b4e3a]: https://twitter.com/sensenet "Twitter"
+  [b175e3d7]: https://www.facebook.com/sensenetcms/ "sensenet on Facebook"
+
+Follow us on [Twitter][0c3b4e3a], like us on [Facebook][b175e3d7], or [clone the repository][bebd2231] and we'll need to check on our stats. We'll hate that (not really, so please, be a good champ).
+
+#### Become a more accessible development platform
```

Comment:
What is the sentence composition you'd like to achieve? 
> Become a more accessible development platform for content repository

maybe? If so, then I would suggest 
> Become a more accessible ECMS development platform